### DISCUSSION TITLE
Had to manually update budgie-control-center in AUR for Arch Linux

### DISCUSSION THREAD

**Discussion by Hi-Phile** [BuddiesOfBudgie/budgie-desktop#84] [General]
Hope this might help others.  I had a problem once the Budgie 10.6 DE was updated.  I am running RebornOS, I ran into an issue where the Budgie task bar wouldn't open the 'Settings' when clicking on the 'Sound', Bluetooth settings, nor the Time/Date. I had to manually replace the gnome-control-center with the budgie-control-center.

`sudo pacman -Rdd gnome-control-center`
then:
`yay -S budgie-control-center`

**Comment by serebit** [comment]
Yeah, Arch doesn't have B-C-C in the repos yet, so you'll have to contend with the AUR for now. There's a bug report on the Arch Linux bug tracker for this: https://bugs.archlinux.org/task/74169?project=5&opened=37021

You can also vote for the budgie-control-center AUR package in order to get inclusion prioritized: https://aur.archlinux.org/packages/budgie-control-center
